doc-content/enterprise-only/installation/patches-applying-proc.adoc

```diff
@@ -6,6 +6,11 @@ Automated update tools are often provided with both patch updates and new minor
 
 You can use the same automated update tool to apply both patch updates and minor release upgrades to {PRODUCT} {PRODUCT_VERSION}. Patch updates of {PRODUCT}, such as an update from version {PRODUCT_VERSION} to {PRODUCT_VERSION}.1, include the latest security updates and bug fixes. Minor release upgrades of {PRODUCT}, such as an upgrade from version 7.7.x to 7.8, include enhancements, security updates, and bug fixes.
 
+[IMPORTANT]
+====
+{PRODUCT} 7.9 provides design and runtime support for DMN 1.2 models at conformance level 3, and runtime-only support for DMN 1.1 and 1.3 models at conformance level 3. Any DMN 1.1 and 1.3 models that you import into {CENTRAL}, open in the DMN designer, and save are converted to DMN 1.2 models. DMN 1.1 and 1.3 models are not supported in the DMN designer in {CENTRAL}.
```

Comment:
Actually, we're not currently converting models into DMN 1.2. Thus, I suggest this sentence (reference: https://github.com/kiegroup/kie-docs/pull/2827):

```suggestion
{PRODUCT} 7.9 provides design and runtime support for DMN 1.2 models at conformance level 3, and runtime-only support for DMN 1.1 and 1.3 models at conformance level 3. You can design your DMN models directly in {CENTRAL} or import existing DMN models into your {PRODUCT} projects for deployment and execution. DMN 1.1 and 1.3 models are currently not supported in the DMN designer in {CENTRAL}.
```

Reply:
@karreiro Thanks. This is based on an existing statement. I'll copy it below. Should this be corrected?

{PRODUCT} provides design and runtime support for DMN 1.2 models at conformance level 3, and includes enhancements and fixes to FEEL and DMN model components to optimize the experience of implementing DMN decision services with {PRODUCT}. {PRODUCT} also provides runtime-only support for DMN 1.1 and 1.3 models at conformance level 3, but any DMN 1.1 models that you import into {CENTRAL}, open in the DMN designer, and save are converted to DMN 1.2 models. DMN 1.3 models are not supported in the DMN designer in {CENTRAL}.

Reply:
@csherrar actually, we need to change this part "**you import into {CENTRAL}, open in the DMN designer, and save are converted to DMN 1.2 models**". We're not converting 1.1 models anymore. This is the temporary limitation we're warning users :)

Reply:
Got it, thanks! Will make those changes now